docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -0,0 +1,74 @@
+
+It is advisable that you call the startProfiling method right after the Sentry SDK is initialized so that you gain visibility at the earliest point in your application lifecycle. From then on, the profiler will keep collecting profiles and sending chunks to Sentry until stopProfiling is called.
+
+## Enabling transaction or continuous profiling mode
+
+Unfortunately, it is currently not possible to use both profiling modes at the same time, as they are mutually exlusive. Since the mode you intend on using depend on the SDK initialization arguments, it means that the profiling mode will have to be selected at the time the Sentry SDK is initialized.
+
+To enable continuous profiling mode, you should ensure that neither profileSampleRate or profilesSampler are set as the values to the Sentry.Init call. If either of those value are set, the SDK will default to transaction based profiling. When the SDK is configured for continuous profiling, the top level calls to start profiles will enable calls to the profiler. If the SDK is configured for transaction based profiling, then these calls will void and not trigger the profiler.
+
+Example of enabling continuous profiling in NodeJS
+
+```javascript
+Sentry.Init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [nodeProfilingIntegration()]
+})
+
+Sentry.profiler.startProfiling();
+```
+
+If you wish to keep using transaction based profiling, then the options remain the same and you should either set profilesSampleRate or profilesSampler option on the SDK.
+
+
+Example of enabling transaction based profiling in NodeJS
+
+```javascript
+Sentry.Init({
+  dsn: "___PUBLIC_DSN___",
+  profileSampleRate: 0.1 // profiles 10% of transactions
+  integrations: [nodeProfilingIntegration()]
+})
+
+const transaction = Sentry.startTransaction();
+//code executed between these two calls is subject to profiling
+transaction.finish();
+```
+
+Note that while the profiling mode cannot be changed at runtime, it is fine for different projects or applications to use different profiling modes, or to switch modes.
+
+## Differences in product experience
+
+The major difference in product experience when using continuous profiling is that you will be able to visualize a flamegraph for your entire application, which means you can now take a step back from the previous transaction based view and look at your application's runtime as a whole. This makes it easier to for you to prioritize the functions that are slowing down your entire application and not just one particular transaction.
+
+The product experience otherwise remains largely the same with entrypoints into profiling being supported from various parts of the performance product.
```

Comment:
```suggestion

## Differences When Using Sentry

The major difference between continuous profiling and transaction-based profiling when using Sentry, is that with continuous profiling you'll be able to visualize a flamegraph for your entire application. This means, that you'll be able to take a step back from the previous transaction-based view and look at your application's runtime as a whole, which makes it easier to prioritize the functions that are slowing down your entire application and not just one particular transaction.

Otherwise, what you'll see in Sentry will be mostly the same. Entry points into profiling will continue to be supported by various parts of the performance product.
```